vagrant.md

```diff
@@ -1,17 +1,17 @@
 # Vagrant
 
-Vagrant is a tool to emulate a cluster of machines.
+Vagrant is a tool to emulate a cluster of machines. Version **2.0.1** is recommended.
 
 ## Requirements
 
-The user needs to have [VirtualBox](https://www.virtualbox.org/wiki/Downloads) installed.
+The user needs to have [VirtualBox](https://www.virtualbox.org/wiki/Downloads) installed. Version **5.2.4** is recommended.
 
 ## Installation
 
 For Linux systems a simple package installation is enough.
 ```
 #Ubuntu
-sudo apt-get install vagrant
+sudo apt-get install vagrant=2.0.1
```

Comment:
This will not work on any Ubuntu releases. The Ubuntu releases ship with older versions of Vagrant. Asking for this latest version should not work. 

You could replace it with
```
wget https://releases.hashicorp.com/vagrant/2.0.1/vagrant_2.0.1_x86_64.deb
sudo dpkg -i vagrant_2.0.1_x86_64.deb
```

Reply:
Very point. I just did the change.